python/lsst/ts/scriptqueue/base_script.py

```diff
@@ -336,13 +336,13 @@ def assert_state(self, action, states):
         * Receive the configuration as a ``yaml`` string.
```

Comment:
Do we have documentation about how we do YAML files yet? If so, we should put links in the appropriate places

Reply:
Good question. These won't be necessarily be files (unlike CSC config), and they need to decode as dicts. I'll say something.